packages/tests/components/SfListItem/SfListItem.PageObject.ts

```diff
@@ -8,6 +8,13 @@ export default class SfListItemObject extends BasePage {
     return this;
   }
 
+  childrenHaveTag(tagName: string) {
+    this.container.then((el) => {
+      expect(el[0].children[0].tagName).to.equal(tagName);
```

Comment:
This test checks if first children's tag is as expected. According to the implementation all 3 children must match this tag. Could you iterate through children and test all of them? I know prefix and suffix are optional, but we should test them too.

[`each`](https://docs.cypress.io/api/commands/each) method may help you.

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2888/commits/63c0c62048b89164de41862f6a610a5acb1f2059